.eslintrc.json

```diff
@@ -19,6 +18,7 @@
   "plugins": ["@typescript-eslint"],
   "rules": {
     "@typescript-eslint/ban-ts-comment": "warn",
-    "react/react-in-jsx-scope": "off"
+    "react/react-in-jsx-scope": "off",
+    "react/jsx-key": "off"
```

Comment:
Why, react do need keys

Reply:
Because of this 
![image](https://github.com/adguernier/ra-ecommerce/assets/7949510/53e4d00e-4505-4e82-a730-eff8f2852f82)

Reply:
Then no, disable the rule inline but not globally otherwise you might miss some actual mistakes